main/acle.md

```diff
@@ -402,6 +402,7 @@ Armv8.4-A [[ARMARMv84]](#ARMARMv84). Support is added for the Dot Product intrin
   a misplaced anchor.
 * Added `__FUNCTION_MULTI_VERSIONING_SUPPORT_LEVEL` to indicate the support
   level of the [Function Multi Versioning](#function-multi-versioning).
+* Unified Function Multi Versioning features sha1, sha2.
```

Comment:
SHA-1 and SHA-2.

(I missed the capitalization on a previous review - please make all mentions of SHA consistent. It's an acronym so must be capitalized.

Reply:
This is referencing the toolchain feature string, not the hash functions themselves, so lowercase is correct. If anything needs to change, it would be to put in quotes or format it as code.